### PR TITLE
Read All Memos

### PR DESCRIPTION
**Pull request by AlphaT3ch**
Allows user to do "/msg MemoServ READ ALL" as requested in [Bug #0001221](http://bugs.anope.org/view.php?id=1221)

### CODE REVIEW

**Comment by del6597** [comment]
Why does it read the memos in reverse order?  NEW reads them incrementally, so why is this different?
It may show you the newest memos first, but in the case of a user sending a large message across many memos, then they will appear out of order.

**Comment by AlphaT3ch** [comment]
Thanks for the feedback, @del6597, I've fixed the error and added it to the fork :+1:

**Comment by JeDa** [comment]
Looks nice :+1:

File: modules/commands/ms_read.cpp [hunk 1]
```diff
@@ -167,6 +167,13 @@ class CommandMSRead : public Command
 				for (i = 0, end = mi->memos->size() - 1; i < end; ++i);
 				MemoListCallback::DoRead(source, mi, ci, i);
 			}
+			else if (numstr.equals_ci("ALL"))
+			{
+				for (i = 0; i < mi->memos->size(); i++)
```
Comment (del6597) [inline modules/commands/ms_read.cpp]: It may be convention to use ```++i``` here, instead of ```i++```.  I guess that's up to @Adam- though.

**Comment by Adam-** [comment]
I guess this is okay. Squash the commits.

**Comment by AlphaT3ch** [comment]
Opened as PR #135

**Comment by Adam-** [comment]
You know you can just force push the branch instead of creating an entire new pr right. That just creates useless spam.

**Comment by AlphaT3ch** [comment]
@Adam- Oh whoops, my bad, sorry

**Pull request closed without merging**